google-cloud-storage/lib/google/cloud/storage/project.rb

```diff
@@ -416,7 +416,8 @@ def create_bucket bucket_name,
                           versioning: nil,
                           requester_pays: nil,
                           user_project: nil,
-                          autoclass_enabled: false
+                          autoclass_enabled: false,
+                          enable_object_retention: nil
```

Comment:
Can you document this param (and necessary example snippets) in the YARD comments for this method?

Reply:
Done.

Reply:
Can you also add some `@example` snippets in the YARD comments?

Reply:
Done.